pkg/api/api.go

```diff
@@ -15,15 +15,21 @@ const (
 	OpenShiftConsoleConfigMapName       = "console-config"
 	OpenShiftConsolePublicConfigMapName = "console-public"
 	ServiceCAConfigMapName              = "service-ca"
-	OpenShiftConsoleDeploymentName      = OpenShiftConsoleName
-	OpenShiftConsoleServiceName         = OpenShiftConsoleName
-	OpenShiftConsoleRouteName           = OpenShiftConsoleName
-	OAuthClientName                     = OpenShiftConsoleName
-	OpenShiftConfigManagedNamespace     = "openshift-config-managed"
-	OpenShiftConfigNamespace            = "openshift-config"
-	OpenShiftCustomLogoConfigMapName    = "custom-logo"
-	TrustedCAConfigMapName              = "trusted-ca-bundle"
-	TrustedCABundleKey                  = "ca-bundle.crt"
-	TrustedCABundleMountDir             = "/etc/pki/ca-trust/extracted/pem"
-	TrustedCABundleMountFile            = "tls-ca-bundle.pem"
+
```

Comment:
Deduplicate `ServingCertSecretName` and `OAuthConfigName` to avoid any other potential bugs.  Noticed these strings have more than one instance.